src/main/kotlin/net/corda/training/flow/IOUSettleFlow.kt

```diff
@@ -71,11 +71,17 @@ class IOUSettleFlow(val linearId: UniqueIdentifier, val amount: Amount<Currency>
 
         // Step 8. Verify and sign the transaction.
         builder.verify(serviceHub)
-        val ptx = serviceHub.signInitialTransaction(builder)
+        val myKeysToSign = cashKeys + ourIdentity.owningKey
```

Comment:
Can you add some comments either here or in the unit tests on what the cashKeys are and how they get signed. IMO this is one of the more confusing parts of flows and we should document thoroughly.